inttest/sonobuoy/terraform/gh.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -e
+sleep 30
+
+echo "prepare environment"
+sudo snap install jq
+sudo snap install kubectl --classic
+
+# Parse CLI args
+readonly git_tag="v0.7.0-beta1"
+readonly github_repo_owner="k0sproject"
+readonly github_repo_name="k0s"
+readonly release_asset_filename="k0s-v0.7.0-beta1-amd64"
```

Comment:
This we should be able to concat based on the version variable

Reply:
I have removed this script entirely